lib/puppet/indirector/file_server.rb

```diff
@@ -15,7 +15,15 @@ def authorized?(request)
 
     # If we're not serving this mount, then access is denied.
     return false unless mount
-    mount.allowed?(request.node, request.ip)
+
+    # If there are no auth directives or there is an 'allow *' directive, then
+    # access is allowed.
+    if mount.empty? || mount.globalallow?
```

Comment:
@MosesMendoza ok, but if we do that, I think it means we have to support / require the use of `allow *` indefinitely - for as long as we have `fileserver.conf` support.  I'd like to get @joshcooper's take on this.